gradle/publish.gradle

```diff
@@ -71,24 +71,6 @@ def pomDevelopers() {
   }
 }
 
-def releaseNotesFileDir = "$project.buildDir/src/common/main/resources"
-task copyReleaseNotesToCommonResourcesDir(type: Copy) {
-  from(rootDir) {
-    include "release_notes.txt"
-  }
-  into releaseNotesFileDir
-}
-
-processResources.dependsOn copyReleaseNotesToCommonResourcesDir
-
-sourceSets {
-  main {
-    resources {
-      srcDir releaseNotesFileDir
-    }
-  }
-}
```

Comment:
I don't see any `release_notes.txt` file anywhere

Reply:
it was old code.. sourceSets is needed though.. it is referenced somewhere further down.

Reply:
it's used prepareSourcesArchive.  release pipeline will fail when sourceSets is removed.

Reply:
I just tested, seems to still work, `main` is a default source set, so already available